6kyuKatas/Your_order_please.py

```diff
@@ -13,4 +13,4 @@ def order(sentence):
 
 # Solution 2
 def order(sentence):
-    return " ".join(sorted(sentence.split(), key=lambda x: int(filter(str.isdigit, x))))
+    return " ".join(sorted(sentence.split(), key=lambda x: list(filter(str.isdigit, x))[0]))
```

Comment:
Good catch! Could you also update the solution on markdown file here? -->  https://github.com/hevalhazalkurt/codewars_python_solutions/blob/master/6kyuKatas/Your_order_please.md